Xamarin.Forms.Platform.iOS/CollectionView/ObservableItemsSource.cs

```diff
@@ -59,6 +59,7 @@ void CollectionChanged(object sender, NotifyCollectionChangedEventArgs args)
 					Move(args);
 					break;
 				case NotifyCollectionChangedAction.Reset:
+					_collectionView.ReloadData();
```

Comment:
failing on IOS10 :(

Reply:
Still not working.. XF Version = 4.8.0.1687